src/core_plugins/tile_map/public/coordinate_maps_visualization.js

```diff
@@ -184,7 +184,7 @@ export function CoordinateMapsVisualizationProvider(Notifier, Private) {
     async getGeohashBounds() {
       const agg = this._getGeoHashAgg();
       if (agg) {
-        const searchSource = this.vis.API.createInheritedSearchSource(this.vis.searchSource);
+        const searchSource = this.vis.searchSource.makeChild();
```

Comment:
The same here as below (yeah sorry for the mixed up order). We'll change the behavior here slightly, by calling the parent searchsource start handler when calling this. The handler will eventually call the `modifyAggConfigOnSearchRequestStart` of the individual aggregations. I am not sure if we want this to happen in this place already (and the one below). It seems to work fine, but I am wondering about some weird situations we might end up with, where we should not call that method "out of place". Maybe we could give the `makeChild` method an options, parameter to pass to the `inherit` method, so we can only enable `callParentStartHandlers` in the one place in courier request handler where it was also enabled beforehand?